packages/office-addin-dev-certs/package.json

```diff
@@ -17,7 +17,14 @@
     "office-addin-dev-certs": "./cli.js"
   },
   "keywords": [
-    "Office"
+    "Office",
+    "SSL",
+    "https",
+    "Certificates",
```

Comment:
Any particular reason for capitalizing Certificates and Self-Signed?

Reply:
updated